pages/deployment/deployment.md

```diff
@@ -42,7 +42,7 @@ specified, it defaults to running `node server.js`.)
 A good example of this is the [text-to-speech][text-to-speech] application -
 here's its `package.json` file*:
 
-```json
+```JSON
```

Comment:
@emirotin not really sure, i just saw it in uppercase in the listing on the highlight.js